meteor/client/ui/Status/StatusCodePill.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
+import * as reacti18next from 'react-i18next'
+import { PeripheralDeviceAPI } from '../../../lib/api/peripheralDevice'
+import { assertNever } from '../../../lib/lib'
+import ClassNames from 'classnames'
+import { StatusCode } from '@sofie-automation/blueprints-integration'
+import { statusCodeToString } from './SystemStatus'
+
+interface StatusCodePillProps {
+	connected: boolean
+	statusCode: StatusCode
+	messages?: string[]
+}
+export const StatusCodePill = reacti18next.withTranslation()(
+	class StatusCodePill extends React.Component<StatusCodePillProps & reacti18next.WithTranslation, {}> {
```

Comment:
Generally, I would suggest using a React Function component here, since this is essentially a bare render function.